src/main/java/net/syscon/elite/api/model/PrivilegeSummary.java

```diff
@@ -26,129 +26,30 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @EqualsAndHashCode
+@ToString
+@Data
 public class PrivilegeSummary {
-    @JsonIgnore
-    private Map<String, Object> additionalProperties;
 
+    @ApiModelProperty(required = true, value = "Offender booking identifier.", example = "112321", position = 1)
     @NotNull
     private Long bookingId;
 
+    @ApiModelProperty(required = true, value = "The current IEP level (e.g. Basic, Standard or Enhanced).", example = "Basic", allowableValues = "Basic,Standard,Enhanced", position = 2)
+    @NotBlank
+    private String iepLevel;
+
+    @ApiModelProperty(required = true, value = "Effective date of current IEP level.", example = "2019-01-24", position = 3)
     @NotNull
     private LocalDate iepDate;
 
+    @ApiModelProperty(value = "Effective date & time of current IEP level.", example = "2019-01-24 15:00:00", position = 4)
     private LocalDateTime iepTime;
 
-    @NotBlank
-    private String iepLevel;
 
+    @ApiModelProperty(required = true, value = "The number of days since last review.", example = "35", position = 5)
     @NotNull
     private Integer daysSinceReview;
 
+    @ApiModelProperty(value = "All IEP detail entries for the offender (most recent first).", position = 6)
     private List<PrivilegeDetail> iepDetails;
-
-    @JsonAnyGetter
```

Comment:
Maybe explain why all these below have gone?

Reply:
Changed to use lombok instead